specs/sb/api.md

```diff
@@ -342,6 +400,38 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
```

Comment:
What are differences between properties and parameters of an instance? I think it may be confusing. What if simplify this to
"schemas":{
  "service_instance":{
    "properties": {},
    "required": []
  }
}